test/tests/functional/pbs_highreslog.py

```diff
@@ -0,0 +1,148 @@
+# later version.
+#
+# PBS Pro is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.
+# See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Commercial License Information:
+#
+# For a copy of the commercial license terms and conditions,
+# go to: (http://www.pbspro.com/UserArea/agreement.html)
+# or contact the Altair Legal Department.
+#
+# Altair’s dual-license business model allows companies, individuals, and
+# organizations to create proprietary derivative works of PBS Pro and
+# distribute them - whether embedded or bundled with other software -
+# under a commercial license agreement.
+#
+# Use of Altair’s trademarks, including but not limited to "PBS™",
+# "PBS Professional®", and "PBS Pro™" and Altair’s logos is subject to Altair's
+# trademark licensing policies.
+
+from tests.functional import *
+
+
+class TestHighResLogging(TestFunctional):
+    """
+    TestSuite for High resolution logging in PBS
+    """
+    tm_micro_re = re.compile(
+        r'(\d{2,2}/\d{2,2}/\d{4,4}\s\d{2,2}:\d{2,2}:\d{2,2}.\d{6,6})')
+
+    def test_disabled(self):
+        """
+        Default High resolution should be disabled (logfile version)
+        """
+        j = Job(TEST_USER)
+        jid = self.server.submit(j)
+        self.server.expect(JOB, {ATTR_state: 'R'}, id=jid)
+        lines = self.server.log_lines(logtype=self.server,
+                                      starttime=self.server.ctime)
+        ok = True
+        for line in lines:
+            m = self.tm_micro_re.match(line)
+            if m is not None:
+                ok = False
+        _msg = 'Found high resolution time stamp in log,' \
+               ' it shouldn\'t be there'
+        self.assertTrue(ok, _msg)
+        _msg = self.server.shortname + \
+            ': High resolution time stamp correctly not in log'
+        self.logger.info(_msg)
+
+    def test_disabled_tracejob(self):
+        """
+        Default High resolution should be disabled (tracejob version)
+        """
+        j = Job(TEST_USER)
+        jid = self.server.submit(j)
+        self.server.expect(JOB, {ATTR_state: 'R'}, id=jid)
+        lines = self.server.log_lines(logtype='tracejob', id=jid, n='ALL')
+        if len(lines) > 1:
+            # Remove first line as it is not log line
+            lines = lines[1:]
+        ok = True
+        for line in lines:
+            m = self.tm_micro_re.match(line)
+            if m is not None:
+                ok = False
+        _msg = 'Found high resolution time stamp in tracejob, ' \
+            'it should not be there'
+        self.assertTrue(ok, _msg)
+        _msg = self.server.shortname + \
+            ': High resolution time stamp correctly not in tracejob output'
```

Comment:
Again, did you mean to say "High resolution time stamp correctly not **set** in tracejob output" ?

Reply:
Yep, same typo.  Thanks for pointing it out.